### ISSUE TITLE
SSL Connection Error / Issues with PR 12

### ISSUE THREAD

**Issue by shofetim** [joy-framework/http#13]
Hello,

Prior to #12 this code worked:

```clojure
(import http)
(pp (http/get "https://example.com"))
```

Post PR #12 it instead errors with "SSL Connection Error"

I believe that this (from #12)
```clojure
Janet janet_port = janet_table_get(options, janet_ckeywordv("port"));
  long port = 80;

  if (janet_checktype(janet_port, JANET_NUMBER)) {
    port = (long)janet_unwrap_integer(janet_port);
  }
```
was intended to supply a default port (80) if no port was specified, which it does, but in so doing it makes port a _required_ option for all non-http requests.

Prior to #12 we could already specify a port, like so:
```clojure
(import http)
(pp (http/get "http://localhost:8080"))
```
Which to me feels like the more natural place to do so.

Could we return to curl determining the default port if not specified?

**Comment by swlkr** [comment]
Oh my gosh, I didn't even realize this, yeah I'll roll it back

**Comment by swlkr** [comment]
Closed by 9c93d468ecb014646a71a45c36598a8cbd1b59f6
